test/test_autograd.py

```diff
@@ -2564,6 +2564,25 @@ def run_test(upper, dims):
         for upper, dims in product([True, False], [(3, 3), (5, 3, 3), (4, 3, 2, 2)]):
             run_test(upper, dims)
 
+    @skipIfNoLapack
+    def test_cholesky_inverse(self):
+        def _test_with_size(upper, dims):
+            # Too small values cause inconsistencies in gradient calculation, hence the special initialization
```

Comment:
Let' be explicit that we need to construct a Cholesky factor here, not just any matrix without "too small values". Might also make sense to allow negative off-diagonal entries in the tests. The issue with the diagonal entries being close to zero is just that the matrix will be almost singular.
Something along the lines of 
```
A_ = torch.randn(*dims)
A_.diagonal().uniform_(0.1, 5)
A_.requires_grad_()
```

I'm using `A_` here because `root` is confusing (the triangular part of it is what's considered the root - i.e. `A` in `func`).

Reply:
Thank you for your comment, I'll fix this.